app/api/groups/get_team_progress.go

```diff
@@ -0,0 +1,99 @@
+package groups
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/go-chi/render"
+	"github.com/jinzhu/gorm"
+
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+func (srv *Service) getTeamProgress(w http.ResponseWriter, r *http.Request) service.APIError {
```

Comment:
Doc/spec missing. You should probably do it while writing the code so that you can add every point you are thinking about when implementing the service.

Reply:
fixed